openedx_lti_tool_plugin/views.py

```diff
@@ -361,4 +391,7 @@ def get(self, request: HttpRequest, course_id: str) -> Union[HttpResponse, HttpR
         if not self.is_user_enrolled(request.user, course_id):
             return HttpResponseForbidden(_(f'{request.user} is not enrolled to {course_id}'))
 
-        return self.render_to_response({'course_outline': self.get_course_outline(request, course_id)})
+        return self.render_to_response({
+            'course_outline': self.get_course_outline(request, course_id),
+            'course_id': course_id
```

Comment:
What do you think about obtaining the course it from course_outline.id on the template instead of adding course_id to the template context?

Reply:
the problem with this is that course_outline.id returns the xblock-like id of the course, not the normal course id.

Reply:
Trailing comma. @alexjmpb